### PR TITLE
Parity config update

### PR DESCRIPTION
**Pull request by holiman**
Handle https://github.com/paritytech/parity-ethereum/pull/9480: 
- Remove eip649 references, and has a list of blockRewards

### CODE REVIEW

File: clients/parity:master/chain.json [hunk 1]
```diff
@@ -4,7 +4,9 @@
     "Ethash": {
       "params": {
         "minimumDifficulty": "0x020000",
-        "blockReward": "0x4563918244F40000",
+        "blockReward": {
+          "0": "0x4563918244F40000",
```
Comment (cheme) [inline clients/parity:master/chain.json]: terminal comma breaks jq for me

File: clients/parity:master/parity.sh [hunk 2]
```diff
@@ -109,8 +109,9 @@ if [ "$HIVE_FORK_BYZANTIUM" != "" ]; then
 #	chainconfig=`echo $chainconfig | jq "setpath([\"params\", \"eip98Transition\"]; \"0x$HIVE_FORK_BYZANTIUM\")"`
 
 	# Ethash params
-	chainconfig=`echo $chainconfig | jq "setpath([\"engine\", \"Ethash\", \"params\", \"eip649Reward\"]; \"0x29A2241AF62C0000\")"`
-	chainconfig=`echo $chainconfig | jq "setpath([\"engine\", \"Ethash\", \"params\", \"eip649Transition\"]; \"0x$HIVE_FORK_BYZANTIUM\")"`
+	#  Set blockreward to 3M for byzantium
+	chainconfig=`echo $chainconfig | jq "setpath([\"engine\", \"Ethash\", \"params\", \"blockReward\",\"0x$HIVE_FORK_BYZANTIUM\"]; \"0x29A2241AF62C0000\")"`
+	# difficulty calculation -- aka bomb delay
```
Comment (cheme) [inline clients/parity:master/parity.sh]: For the bomb delay we also need to define additional variables : 
```
chainconfig=`echo $chainconfig | jq "setpath([\"engine\", \"Ethash\", \"params\", \"difficultyBombDelays\",\"0x$HIVE_FORK_BYZANTIUM\"]; \"3000000\")"`
chainconfig=`echo $chainconfig | jq "setpath([\"engine\", \"Ethash\", \"params\", \"difficultyBombDelays\",\"0x$HIVE_FORK_CONSTANTINOPLE\"]; \"5000000\")"`

```

Comment (holiman) [reply clients/parity:master/parity.sh]: I'm not sure that'll make a difference -- on hive there are only a couple of blocks, so whether the bomb is delayed to 3M or 5M shouldn't matter. Hive can't really be used to test the bomb delay, I think

File: clients/parity:master/chain.json (hunk 1 above)
Comment (holiman) [reply clients/parity:master/chain.json]: Fixed, pushed directly into master

File: clients/parity:master/parity.sh (hunk 2 above)
Comment (holiman) [reply clients/parity:master/parity.sh]: Also added an empty difficultyBombDelays struct to chain.json

Comment (cheme) [reply clients/parity:master/parity.sh]: Hello, I made a small (big) mistake, our 'difficultyBombDelays' is additive so the right code should be 
```
chainconfig=`echo $chainconfig | jq "setpath([\"engine\", \"Ethash\", \"params\", \"difficultyBombDelays\",\"0x$HIVE_FORK_BYZANTIUM\"]; \"3000000\")"`
chainconfig=`echo $chainconfig | jq "setpath([\"engine\", \"Ethash\", \"params\", "difficultyBombDelays\",\"0x$HIVE_FORK_CONSTANTINOPLE\"]; \"2000000\")"`
```
to have 5M total delay at constantinople.

**Pull request merged**